src/tmpfiles/tmpfiles.c

```diff
@@ -1562,9 +1561,9 @@ static int create_directory_or_subvolume(Item *i, const char *path) {
                 if (r == -ENOTTY)
                         log_debug_errno(r, "Couldn't adjust quota for subvolume \"%s\" (unsupported fs or dir not a subvolume): %m", i->path);
                 else if (r == -EROFS)
-                        log_debug_errno(r, "Couldn't adjust quota for subvolume \"%s\" (fs is read-only).", i->path);
+                        log_debug("Couldn't adjust quota for subvolume \"%s\" (fs is read-only).", i->path);
```

Comment:
why this change? we should always propagate error codes we get to the journal, and that's what log_debug_errno() does